docs/topics/auth/passwords.txt

```diff
@@ -173,10 +216,32 @@ default PBKDF2 algorithm:
             'django.contrib.auth.hashers.CryptPasswordHasher',
         ]
 
-
 That's it -- now your Django install will use more iterations when it
 stores passwords using PBKDF2.
 
+Argon2
+~~~~~~
+
+Argon2 has three attributes that can be customized:
+
+#. ``time_cost`` controls the number of iterations within the hash
+#. ``memory_cost`` controls the size of memory that must be used during the
+   computation of the hash
+#. ``parallelism`` controls how many CPUs the computation of the hash can be
+   parallelized on.
+
+The default values of these attributes are probably fine for you.  If you
```

Comment:
The defaults aren’t too likely to change very frequently since even the lowest settings are “safe enough”.  Choosing the target verification time is highly individual so I guess the best approach for Django is to decide on a time and tune all checkers to meet it so people don't have to think about it?  My own target was roughly 0.5ms–1ms “on my notebook”.